sklearn/utils/_estimator_html_repr.py

```diff
@@ -119,7 +119,7 @@ def _write_label_html(
             '<input class="sk-toggleable__control sk-hidden--visually"'
             f' id="{est_id}" '
             f'type="checkbox" {checked_str}><label for="{est_id}" '
-            f'class="{label_class} {fitted_str}">{name}'
+            f'class="{label_class} {fitted_str}">&nbsp;&nbsp;{name}'
```

Comment:
why do we need the non breaking spaces?

Reply:
we add "?" and "i" characters that makes the name uncentered.
Non-breaking space are allowing to recenter.

This is just a temporary check. I need to make sure that we have the right number of space depending of the number of information that we add.

And it would also require a comment to explain why :)